store/cluster.go

```diff
@@ -78,6 +78,10 @@ func NewCluster(name string, nodes []string, replicas int) (*Cluster, error) {
 }
 
 func (cluster *Cluster) Clone() *Cluster {
+	if cluster == nil {
```

Comment:
I prefer to check before any method call, e.g. `if c != nil { c.Clone() }`.

Reply:
Yes, it will be better to keep consistent with other methods. Done.